charts/rancher-monitoring/100.2.0+up40.1.2/app-README.md

```diff
@@ -14,6 +14,21 @@ The chart installs the following components:
 
 For more information, review the Helm README of this chart.
 
+## Upgrading from 100.1.0+up19.0.3 to 100.2.0+up40.1.2
+
+### Rebased the chart to upstream version 40.1.2. 
+### Noticeable changes:
+Grafana: 
+- switched from AngularJS to React
```

Comment:
all the bullet points should start with a capitalized word, also is this necessary to mention? Is there any noticeable difference?

Reply:
the nginx config needed to be updated, but I guess that's handled by Nick's PR so we don't really need to mention it (I was thinking it might be useful for debugging purposes)